docs/Migration-from-1.1-to-1.2.md

```diff
@@ -0,0 +1,92 @@
+# Migration from 1.1.x to 1.2.x
+
+This guide contains instructions for migration from PowerAuth Mobile SDK version `1.1.x` to version `1.2.x`.
+
+## Introduction
+
+PowerAuth Mobile SDK in version `1.2.0` introduces following important changes: 
+
+- **Android:** Complete reworked support for biometric authentication.
+- **iOS & Android:** Removes classes related to password strength testing (`PasswordUtil` and `PA2PasswordUtil`)
+
+### Compatibility with PowerAuth Server
+
+- This release is fully compatible with PowerAuth Server version `0.22.x`.
+- If your application doesn't use Recovery Codes, then you can use this SDK also against `0.21.x` servers.
+
+## Android
+
+### API changes
+
+- Removed deprecated `PasswordUtil`, 
+  - We recommend you to use our [Passphrase Meter](https://github.com/wultra/passphrase-meter) library as a replacement.
+  - The removed interface is still available in [0.20.x branch](https://github.com/wultra/powerauth-mobile-sdk/tree/release/0.20.x/proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/util/password), so you can copy that files directly into your project.
+
+- Removed ALL classes from package `io.getlime.security.powerauth.keychain.fingerprint`
+  - All biometry related classes are now located in package `io.getlime.security.powerauth.biometry`
+
+- New biometric authentication implementation now uses different system interfaces, depending on the system version:
+  - `FingerprintManager` is used on Android 6.0, up to 8.1. In this case, our SDK displays a legacy fingerprint dialog (see [Behavioral Changes](#behavioral-changes))
+  - `BiometricPrompt` is used on Android 9.0 and higher
+  - We guarantee, that migration from older PowerAuth Mobile SDKs works properly on Android 9.0+.
```

Comment:
I don't like formulation "we guarantee". I would be careful in the Android world. It works well, but Android is a wild world full of unexpected devices. 
Maybe some better formulation stating that fingerprint auth done with older PA SDK works with newer PA SDK using BiometricPrompt on Android 9.0+.

Reply:
Right 👍